src/_7_21_1_Hermite_Polynomials.jl

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env julia
+#GSL Julia wrapper
+#(c) 2013 Jiahao Chen <jiahao@mit.edu>
+##############################
+# 7.21.1 Hermite Polynomials #
+##############################
+export sf_hermite_prob, sf_hermite_prob_e, sf_hermite_prob_array,
+       sf_hermite_prob_series, sf_hermite_prob_series_e, sf_hermite_phys,
+       sf_hermite_phys_e, sf_hermite_phys_array, sf_hermite_phys_series,
+       sf_hermite_phys_series_e
+
+
+# These routines evaluate the probabilists’ Hermite polynomial He_n(x) of order
+# n at position x.
+#
+#   Returns: Cdouble
+function sf_hermite_prob(n::Integer, x::Real)
+    ccall( (:gsl_sf_hermite_prob, libgsl), Cdouble, (Cint, Cdouble), n, x )
+end
+Compat.@dep_vectorize_2arg Number sf_hermite_prob
```

Comment:
New functions shouldn't have this.

Reply:
yuyichao, what exactly do you mean with "New functions shouldn't have this."
Are you referring to the Compat line behind the functions?!
As far as I understand their job is to ensure compatibility with older versions of julia. As I said I simply copied them from other wrappers in the library. However if they are not necessary anymore I will remove them, of course.

Reply:
> Are you referring to the Compat line behind the functions?!

Yes.

> As far as I understand their job is to ensure compatibility with older versions of julia.

No. It's for compatibility with users that haven't upgraded as explained in the doc of the Compat package https://github.com/JuliaLang/Compat.jl#new-macros

> As I said I simply copied them from other wrappers in the library.

I know that's why I mentioned "New functions".

Reply:
Ok, thanks a lot for the clearification!
I will update the file accordingly.